### DISCUSSION TITLE
Best practice for shareable watch faces

### DISCUSSION THREAD

**Discussion by nlapenn** [joeycastillo/Sensor-Watch#123] [Q&A]
The section about writing watch faces and contributing to community firmware isn't complete yet but I was wondering if there are any tips/tricks/best practices I can do now to make the watch face I'm developing more easily shareable in the future?

**Comment by wryun** [comment]
If you haven't found them already, I'd start with:
- running https://github.com/joeycastillo/Sensor-Watch/blob/main/movement/template/watch_face.py to add a new face
- then follow https://github.com/joeycastillo/Sensor-Watch/tree/main/movement or https://www.sensorwatch.net/docs/movement/newface/

Once you have a face, it's just a matter of making a PR on this repository so it's easier for anyone to use.
